users/views.py

```diff
@@ -21,17 +21,21 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 '''
-from django.shortcuts import render
 import json
 import random
 from index.views import custom_proc
+from general_tools.log import get_logger
 from django.template import RequestContext
+from django.contrib.auth import authenticate, login, logout
+from django.shortcuts import render, redirect
+from users.admin import UserCreationForm, AuthenticationForm
 # Create your views here.
 
+logger = get_logger()
 
 def submit(request):
     return render(request, 'users/submit.html', {},
-                context_instance = RequestContext(request, processors = [custom_proc]))
+                context_instance=RequestContext(request, processors=[custom_proc]))
```

Comment:
according to the [style guide](https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Whitespace#Whitespace), there shouldn't be any space after a keyword.